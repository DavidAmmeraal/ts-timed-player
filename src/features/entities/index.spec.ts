/**
 * index.spec.ts
 */
import * as exported from '.';

describe('entities feature module', () => {
  it('should expose according to snapshot', () => {
    expect(exported).toMatchSnapshot();
  });
});
