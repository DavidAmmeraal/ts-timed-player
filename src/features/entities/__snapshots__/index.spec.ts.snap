// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entities feature module should expose according to snapshot 1`] = `
Object {
  "entitiesActions": Object {
    "createEntityAction": [Function],
    "deleteEntityAction": [Function],
    "updateEntityAction": [Function],
  },
  "entitiesReducer": [Function],
}
`;
